lib/ansible/modules/system/timedatectl.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2018, Susant Sahani <susant@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: timedatectl
+short_description: Automates timedatectl
+description:
+    - Allows you to configure timedated.
+version_added: "2.8"
+options:
+    command:
+        description:
+            - Specifies the command that should be applied.
+        choices: [ "set-time", "set-timezone", "set-local-rtc", "set-ntp" ]
```

Comment:
we normally prefer establishing a 'state' not passing commands

Reply:
Thanks updated.